### PR TITLE
Add Traefik, Wallabag, Olivetin

### PR DESCRIPTION
**Pull request by meonkeys**
## Description

This adds support for Traefik, Wallabag, and Olivetin cards. My motivation is so I can use this with my own services. This adds no new dependencies. Let me know if you'd rather this was in three separate PRs or whatever.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I've read & comply with the [contributing guidelines](https://github.com/bastienwirtz/homer/blob/main/CONTRIBUTING.md)
- [ ] I have tested my code for new features & regressions on both mobile & desktop devices, using the latest version of major browsers. **I did _some_ testing, but not that much**
- [ ] I have made corresponding changes to the documentation (README.md). **N/A**
- [x] I've checked my modifications for any breaking changes, especially in the `config.yml` file

### CODE REVIEW

File: src/components/services/Olivetin.vue [hunk 1]
```diff
@@ -0,0 +1,84 @@
+<template>
+  <Generic :item="item">
+    <template #content>
+      <p class="title is-4">{{ item.name }}</p>
+      <p class="subtitle is-6">
+        <template v-if="versionstring">
```
Comment (bastienwirtz) [inline src/components/services/Olivetin.vue:6]: Could you add a check on `item.subtitle` to display the subtitle insted of the version if set ? (most service card integration works like that, I forgot to mention it on gitea)

**Review by bastienwirtz — CHANGES_REQUESTED** [review]
Awesome! Thanks for your work :pray: 

Could you add an example of the api output here: https://github.com/bastienwirtz/homer/tree/main/dummy-data (more detail in the directory readme) ?

File: src/components/services/Olivetin.vue (hunk 1 above)
Comment (meonkeys) [reply src/components/services/Olivetin.vue]: Sure thing. Done in 1864e1d8253bbfef94ae732defb48ef3ff833750 . Except gitea... I'll clean that up in a separate PR, I guess?

**Comment by meonkeys** [comment]
> Awesome! Thanks for your work 🙏

Of course!

> Could you add an example of the api output

Will do.

**Review by bastienwirtz — APPROVED** [review]

**Pull request merged**